lib/ukdebug/Makefile.uk

```diff
@@ -3,14 +3,20 @@ $(eval $(call addlib_s,libukdebug,$(CONFIG_LIBUKDEBUG)))
 CINCLUDES-$(CONFIG_LIBUKDEBUG)   += -I$(LIBUKDEBUG_BASE)/include
 CXXINCLUDES-$(CONFIG_LIBUKDEBUG) += -I$(LIBUKDEBUG_BASE)/include
 
-LIBUKDEBUG_CFLAGS-y   += -D__IN_LIBUKDEBUG__
-LIBUKDEBUG_CXXFLAGS-y += -D__IN_LIBUKDEBUG__
+LIBUKDEBUG_CFLAGS-y   += -D__IN_LIBUKDEBUG__ -I$(LIBUKDEBUG_BASE)
```

Comment:
Includes should probably go to
`LIBUKDEBUG_CINCLUDES-y` and `LIBUKDEBUG_CXXINCLUDES-y`